### PR TITLE
fix(docs): Reword welcome page to be more friendly

### PR DESCRIPTION
**Pull request by gabrielg42**

### CODE REVIEW

**Review by samiralavi — APPROVED** [review]
Thanks @gabrielg42 all is good

**Comment by gabrielg42** [comment]
@jatinriverlane @safeamiiir Is this ok with either of you?

**Review by safeamiiir — APPROVED** [review]
Thanks Gabriel!

File: docs/index.md [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+In this current version of Aqueduct, we provide a user-friendly GUI and backend for storing, browsing, and searching for experiment results. 
+Your experiment data can be saved into Aqueduct with simple Python APIs which do not disrupt existing workflows. 
+We provide easy sorting and filtering of experiments, and quick previews of text and plots so you can easily find the data that you need. 
+Aqueduct is easy to try and easy to adopt.
+
+In the coming months we will add support for multiple users, automation, and plug-ins.
```
[the remark this answers is not in the document]
Comment (gabrielg42) [reply docs/index.md:13]: Good idea.

**Pull request merged**